tests/testthat/test-github.R

```diff
@@ -0,0 +1,45 @@
+test_that("use_github_links populates empty URL field", {
```

Comment:
Similar tests start with:

```
skip_on_cran()
skip_if_no_git_user()
local_interactive(FALSE)
```

I'm not sure there's any reason to skip on CRAN, if you're mocking the GitHub calls, so maybe that's not needed.

`skip_if_no_git_user()` is needed if `use_git()` stays (some CRAN machines don't have git or a git user? anyway, this skip exists for a reason 😬 ).

`local_interactive(FALSE)` is just something I leave behind in tests where that is helpful during interactive tinkering.